Sources/KlaviyoCore/Vendor/ReachabilitySwift.swift

```diff
@@ -48,16 +47,39 @@ func callback(reachability:SCNetworkReachability, flags: SCNetworkReachabilityFl
     }
 }
 
-class Reachability {
-
-    typealias NetworkReachable = (Reachability) -> ()
-    typealias NetworkUnreachable = (Reachability) -> ()
+public class Reachability {
```

Comment:
Was this all from linting?

Reply:
Sorry it's a little confusing. I updated `Reachability` and certain properties within it as well we use this in something that KlaviyoSwift uses like, 

```swift 
public enum LifeCycleEvents {
    case terminated
    case foregrounded
    case backgrounded
    case reachabilityChanged(status: Reachability.NetworkStatus)
}
```

Since I made this change, I think lint formatting the entire file.